plugins/linkattacher/src/linkattacherinterface.cpp

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright (C) 2013-2015 Fondazione Istituto Italiano di Tecnologia RBCS & iCub Facility & ADVR
+ * Authors: see AUTHORS file.
+ * CopyPolicy: Released under the terms of the LGPLv2.1 or any later version, see LGPL.TXT or LGPL3.TXT
+ */
+
+#include <linkattacherinterface.h>
+
+using namespace gazebo;
+using namespace std;
+using namespace yarp::os;
+
+LinkAttacherInterface::LinkAttacherInterface() : ModelPlugin(), m_network(0), m_rpcport(0)
+{
+}
+
+LinkAttacherInterface::~LinkAttacherInterface()
+{
+  if(m_rpcport)
+  {
+    m_rpcport->close();
+    delete m_rpcport;
+  }
+
+  if(m_network)
```

Comment:
Brackets are always a good pattern even for single line `if`

Reply:
done

Reply:
Another suggestion about conditional and looping statements. it was suggested to me by @traversaro that we could use a space after `if` or `for` , etc. for example `if (condition)` instead of  `if(conditon)`

Although this is trivial, it helps us differentiate between other regular functions calls. Also makes the code look neat and presentable.

Reply:
@prashanthr05 If thinks @francesco-romano will be crying as soon as he sees your comment.

Reply:
🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉 🎉